test/README.md

```diff
@@ -26,12 +26,48 @@ Make sure you have valid credentials for accessing GCP:
 gcloud auth login
 ```
 
+Set the project property in the core section by running:
+```
+gcloud config set project PROJECT_ID
+```
+
+Installed jq by running.
+```
+sudo apt-get install jq
+```
+
 Make sure you are at the root directory of the repository.
 
 ```
 cd gke-networking-recipes
 ```
 
+
+First, set ZONE and SUBNET_REGION environment variable, then run ./test/generate_config.sh to generate the testing config.
+```
+export ZONE=zone
+export SUBNET_REGION=subnet_region
+
+./test/generate_config.sh
+```
+
+To run ingress-https test, you need a special project($DNS_PROJECT) that has configured DNS domain, and make sure the current service account have role/dns_admin access to that project. 
```

Comment:
These setting should be local to the test itself, not part of the generic settings.